Series.txt

```diff
@@ -321,10 +321,9 @@
 	ID 164
 0 NASCAR iRacing Class A : NASCAR Class A
 0 NASCAR iRacing Class A Fixed : NASCAR Class A Fixed
-4115W6 NASCAR iRacing Class B : Oval
+4115W2 NASCAR iRacing Class B : Oval
```

Comment:
W6 => W2, I'm basically guessing. I have no idea what this does.

Reply:
"W6" means "Week 6".

Reply:
It gets auto-updated on the server as the weeks change (but not auto-committed to GitHub)...

Reply:
ah, ok, so maybe this was just missed during a manual update? Everything else is listed as week 2.

This PR is probably still worth merging since it has the logo change.